AmplifyPlugins/Predictions/AWSPredictionsPlugin/Support/Internal/ErrorHandling/Rekognition+PredictionsErrorConvertible.swift

```diff
@@ -0,0 +1,121 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import AWSRekognition
+import Amplify
+import ClientRuntime
+
+
+extension AWSRekognition.HumanLoopQuotaExceededException: PredictionsErrorConvertible {
+    var predictionsError: PredictionsError {
+        .service(
+            .init(
+                description: "",
```

Comment:
is this intended to have no description or recovery suggestion?

Reply:
Thanks for catching that. These were previously without description / recovery suggestion as well. Addressed in [f13a59a](https://github.com/aws-amplify/amplify-swift/pull/3257/commits/f13a59a4dd5e7767cc614347a072aaa41ec81089)